src/components/proofs/EmailProofForm.tsx

```diff
@@ -35,6 +36,7 @@ export default function ({
 }) {
   const [loading, setLoading] = useState(false)
   const [email, setEmail] = useState<string>('')
+  const { xxs } = useBreakpoints()
```

Comment:
It will require additional styling, should we do this?
![image](https://user-images.githubusercontent.com/49658988/192801833-4d58bc63-e924-4df6-ad98-21178a9d2c41.png)
Without this stuff it will looks like this:
![image](https://user-images.githubusercontent.com/49658988/192801954-b46e56e5-31e6-492a-a749-c5504711c700.png)